draw2dimg/ftgc.go

```diff
@@ -58,6 +58,8 @@ func NewGraphicContext(img draw.Image) *GraphicContext {
 	switch selectImage := img.(type) {
 	case *image.RGBA:
 		painter = raster.NewRGBAPainter(selectImage)
+	case *image.Gray:
+		painter = raster.NewGRAYPainter(selectImage)
```

Comment:
Also, I don't know draw2d very well, but you might want to watch out that NewRGBAPainter uses the draw.Over composition operator, from the image/draw standard library, by default. In comparison, the NewGrayPainter proposed at https://github.com/golang/freetype/pull/41 in effect uses the draw.Src composition operator.

Your GrayPainter's Paint method also ignores the Span.Alpha field, which looks like a mistake.